build.rs

```diff
@@ -77,7 +77,11 @@ fn write_cpp_conf_pri(path: &Path) -> Result<(), Box<dyn Error>> {
     let mut f = fs::File::create(path)?;
 
     writeln!(&mut f)?;
-    writeln!(&mut f, "INCLUDEPATH = /usr/local/include")?;
+    if let Ok(boost_path) = get_boost_path() {
```

Comment:
This looks like it will ignore errors?

Reply:
ooh sorry! wanted to write a fancy clause, not realizing the full effect. 
fix is in